lib/common.sh

```diff
@@ -159,6 +159,9 @@ else
   export IPAMBRANCH="${IPAMBRANCH:-main}"
 fi
 
+export CAPM3BRANCH="${CAPM3BRANCH:-peppi-lotta/fix/Update-Kustomize-depricated-syntax}"
```

Comment:
This kind of changes are better off put in `config_example.sh` where its clear they're not part of the intended change.

Also, mark it as WIP and add a description.